### DISCUSSION TITLE
Duplicate data definitions found in autodoc

### DISCUSSION THREAD

**Discussion by debin-ge** [Kong/kong#9750] [Help]
In the process of reading the kong source code, I found that there is a duplicate kong.configuration definitions in the openapi-gen.lua file.
<img width="564" alt="image" src="https://user-images.githubusercontent.com/21329997/201303327-3bda292d-9b91-445e-b6ae-3600da4faa3a.png">
I don't understand, is this part of the code duplicated, or there is some other logic?
The directory where the file is located in kong/autodoc/admin-api/openapi-gen.lua

**Comment by StarlightIbuki** [comment]
Thanks for this finding. Would you create a PR for this? We encourage people to contribute to Kong and have a gift for any new contributors.
